platform/mellanox/rules.mk

```diff
@@ -51,4 +51,7 @@ $(DOCKER_PLATFORM_MONITOR)_DEPENDS += $(APPLIBS) $(SX_COMPLIB) $(SXD_LIBS) $(SX_
 # Force the target bootloader for mellanox platforms to grub regardless of arch
 TARGET_BOOTLOADER = grub
 
+# location for the platform specific external kernel patches tarball
+override EXTERNAL_KERNEL_PATCH_TAR := $(BUILD_WORKDIR)/$(PLATFORM_PATH)/non-upstream-patches/patches.tar.gz
```

Comment:
Is the `override` keyword needed here, or can a regular assignment just work?

Reply:
I guess, `EXTERNAL_KERNEL_PATCH_TAR = $(BUILD_WORKDIR)/$(PLATFORM_PATH)/non-upstream-patches/patches.tar.gz` this'll work as well. But i thought, override here would make it clear that this is the final value. I can change it to a regular assignment if you'd prefer

Reply:
Considering it's in a platform-specific file, I'm fine with it with an `override`. If needed, it can be removed later.